package.json

```diff
@@ -74,7 +74,7 @@
     "distsrc": "sed -ibak 's#\"main\": \"dist/index.js#\"main\": \"src/index.js#' packages/node_modules/{*,@ciscospark/*}/package.json && rimraf packages/node_modules/{*,@ciscospark/*}/*bak",
     "srcdist": "sed -ibak 's#\"main\": \"src/index.js#\"main\": \"dist/index.js#' packages/node_modules/{*,@ciscospark/*}/package.json && rimraf packages/node_modules/{*,@ciscospark/*}/*bak",
     "publish:docs": "grunt --gruntfile Gruntfile.concurrent.js publish:docs",
-    "get-next-version": "babel-node ./tooling/index.js version next --always",
+    "get-next-version": "babel-node ./tooling/index.js version next --always --includeSamples",
```

Comment:
Is there a case where we wouldn't want to include samples in the version calculation?

Reply:
I have no idea, but I wanted to make it configurable